test/resources/rbac.go

```diff
@@ -36,7 +36,10 @@ func AssertServiceAccount(t *testing.T, clients *utils.Clients, ns, targetSA str
 	t.Helper()
 
 	err := wait.Poll(Interval, Timeout, func() (bool, error) {
-		saList, err := clients.KubeClient.Kube.CoreV1().ServiceAccounts(ns).List(context.TODO(), metav1.ListOptions{})
+		saList, err := clients.KubeClient.CoreV1().ServiceAccounts(ns).List(context.TODO(), metav1.ListOptions{})
```

Comment:
i like this change. 
But how did this happen, is `Kube` an embedded object in `KubeClient`

Reply:
ya could be
when dependency got updated this got removed